### PR TITLE
add example pipe stream in c++, python

### PR DESCRIPTION
**Pull request by jshen2036**

### CODE REVIEW

**Comment by yijiew** [comment]
@wes-b and @rabbitdaxi : could you help to review Jingjing's pull request here. It is about piping the sensor data by a cpp server and use python to consume those data. It is more related to the sensor SDK, so I think it might be better for you two to approve the code.

File: pipe-to-python-samples/pipeClientReadImages.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    # Create pipe client
+    fileHandle = win32file.CreateFile("\\\\.\\pipe\\mynamedpipe",
+        win32file.GENERIC_READ | win32file.GENERIC_WRITE,
+        0, None,
+        win32file.OPEN_EXISTING,
+        0, None)
+
+    # For visualization
+    cv2.namedWindow('vis', cv2.WINDOW_NORMAL)
+    cv2.resizeWindow('vis', 1300, 600)
```
Comment (rabbitdaxi) [inline pipe-to-python-samples/pipeClientReadImages.py]: may be  FRAME_WIDTH * 2, FRAME_HEIGHT?

File: pipe-to-python-samples/pipeClientReadImages.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        request_msg = "Request depth image and ir image"
+        win32file.WriteFile(fileHandle, request_msg.encode())
+        # Read reply data, need to be in same order/size as how you write them in the pipe server in pipe_streaming_example/main.cpp
+        depth_data = win32file.ReadFile(fileHandle, FRAME_WIDTH * FRAME_HEIGHT * BYTES_PER_PIXEL)
+        ab_data = win32file.ReadFile(fileHandle, FRAME_WIDTH * FRAME_HEIGHT * BYTES_PER_PIXEL)
+        # Reshape for image visualization
+        depth_img_full = np.frombuffer(depth_data[1], dtype=np.uint16).reshape(FRAME_HEIGHT, FRAME_WIDTH).copy()
+        ab_img_full = np.frombuffer(ab_data[1], dtype=np.uint16).reshape(FRAME_HEIGHT, FRAME_WIDTH).copy()
+        
+        depth_vis = (plt.get_cmap("gray")(depth_img_full/8000.0)[..., :3]*255.0).astype(np.uint8)
```
Comment (rabbitdaxi) [inline pipe-to-python-samples/pipeClientReadImages.py]: >8000.0 [](start = 57, length = 6)

define this at top and add comment

File: pipe-to-python-samples/pipeClientReadImages.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        ab_vis = (plt.get_cmap("gray")(ab_img_full/512.0)[..., :3]*255.0).astype(np.uint8)
```
Comment (rabbitdaxi) [inline pipe-to-python-samples/pipeClientReadImages.py]: >512.0 [](start = 51, length = 5)

define this at top and add comment

File: pipe-to-python-samples/pipe_streaming_example.sln [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+		{AE82EE82-2782-474F-98A9-06CFC7A3BB4D}.Release|x64.ActiveCfg = Release|x64
+		{AE82EE82-2782-474F-98A9-06CFC7A3BB4D}.Release|x64.Build.0 = Release|x64
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {4F4A999C-0964-493D-AB98-112DCD166E33}
+	EndGlobalSection
+EndGlobal
```
Comment (rabbitdaxi) [inline pipe-to-python-samples/pipe_streaming_example.sln]: nit: untabtify

File: pipe-to-python-samples/packages.config [hunk 5]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Kinect.Sensor" version="1.1.0" targetFramework="native" />
```
Comment (rabbitdaxi) [inline pipe-to-python-samples/packages.config]: >"1.1.0" [](start = 54, length = 7)

update this to 1.2.0

File: pipe-to-python-samples/pipe_streaming_example.vcxproj [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+  <ImportGroup Label="ExtensionTargets">
+    <Import Project="packages\Microsoft.Azure.Kinect.Sensor.1.1.0\build\native\Microsoft.Azure.Kinect.Sensor.targets" Condition="Exists('packages\Microsoft.Azure.Kinect.Sensor.1.1.0\build\native\Microsoft.Azure.Kinect.Sensor.targets')" />
+  </ImportGroup>
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('packages\Microsoft.Azure.Kinect.Sensor.1.1.0\build\native\Microsoft.Azure.Kinect.Sensor.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Microsoft.Azure.Kinect.Sensor.1.1.0\build\native\Microsoft.Azure.Kinect.Sensor.targets'))" />
+  </Target>
+</Project>
```
Comment (rabbitdaxi) [inline pipe-to-python-samples/pipe_streaming_example.vcxproj]: update SDK to 1.2.0

Comment (jshen2036) [reply pipe-to-python-samples/pipe_streaming_example.vcxproj]: Updated

File: pipe-to-python-samples/pipeClientReadImages.py (hunk 1 above)
Comment (jshen2036) [reply pipe-to-python-samples/pipeClientReadImages.py]: Updated.

File: pipe-to-python-samples/pipeClientReadImages.py (hunk 2 above)
Comment (jshen2036) [reply pipe-to-python-samples/pipeClientReadImages.py]: Updated.

File: pipe-to-python-samples/packages.config (hunk 5 above)
Comment (jshen2036) [reply pipe-to-python-samples/packages.config]: Updated.

File: pipe-to-python-samples/pipe_streaming_example.vcxproj (hunk 6 above)
Comment (jshen2036) [reply pipe-to-python-samples/pipe_streaming_example.vcxproj]: Updated

File: pipe-to-python-samples/pipe_streaming_example.sln (hunk 4 above)
Comment (jshen2036) [reply pipe-to-python-samples/pipe_streaming_example.sln]: Resolved

File: pipe-to-python-samples/pipeClientReadImages.py (hunk 3 above)
Comment (jshen2036) [reply pipe-to-python-samples/pipeClientReadImages.py]: Updated.

**Pull request merged**